packages/frontend-api/src/Model/Error/ErrorCodeSubscriber.php

```diff
@@ -32,6 +34,11 @@ public function onErrorFormatting(ErrorFormattingEvent $event): void
             $code = $error->getCode();
         }
 
+        if ($error instanceOf Exception && $error->getMessage() === 'Access denied to this field.') {
```

Comment:
I do not like the fact that we rely on the error message here but I understand there is nothing else that we can check :frowning_face:

Reply:
Not FIXED